base/io.jl

```diff
@@ -292,7 +292,7 @@ The text is assumed to be encoded in UTF-8.
 # Examples
 ```jldoctest
 julia> open("my_file.txt", "w") do io
-           write(io, "JuliaLang is a GitHub organization.\nIt has many members.\n");
+           write(io, "JuliaLang is a GitHub organization.\\nIt has many members.\\n");
```

Comment:
Do you intend for this to print out a literal backslash and then the letter `n`?

Reply:
https://docs.julialang.org/en/latest/stdlib/io-network/#Base.readline it looks pretty bad now?

Reply:
Oh, I see. Yes, this is inside a doc string so it's splicing the newline into the doc string. Hmm. It seems a bit unfortunate that we have to double-escape every string literal in docstrings...